web-registry/src/components/organisms/BridgedEcocreditsTable/BridgedEcocreditsTable.constants.ts

```diff
@@ -1,11 +1,15 @@
 export const STATUS_PENDING = 'pending';
 export const STATUS_COMPLETE = 'complete';
 export const STATUS_ERROR = 'error';
+export const TX_STATUS_REFRESH_INTERVAL = 10000;
+export const ONE_MINUTE_MS = 60000;
+export const TWO_MINUTE_MS = ONE_MINUTE_MS * 2;
 
 export const BRIDGED_STATUSES = {
   regen_hash_not_found: STATUS_PENDING,
   regen_ready: STATUS_PENDING,
   evm_broadcast: STATUS_PENDING,
+  evm_have_height: STATUS_PENDING,
```

Comment:
We were missing this status before.

Reply:
good catch, just got added a few days ago actually https://github.com/rnd-inc/toucan-bridge-service/pull/303